test/run_tests.py

```diff
@@ -211,6 +211,45 @@ def test_inference():
     assert_dir("model_output/llik")
     # os.removedirs("model_output")
 
+def test_inference1():
+    os.chdir("test_inference1")
```

Comment:
similarly let's change the folder name to match the test

Reply:
Yes, I can make the name changes everywhere. Once we have the name finalized